src/commands/app/init.js

```diff
@@ -28,9 +34,30 @@ class InitCommand extends BaseCommand {
 
     aioLogger.debug('creating new app with init command ', flags)
 
+    // default project name and services
     let projectName = path.basename(process.cwd())
     let services = 'AdobeTargetSDK,AdobeAnalyticsSDK,CampaignSDK,McDataServicesSdk,AudienceManagerCustomerSDK' // todo fetch those from console when no --import
 
+    if (!(flags.import || flags.yes)) {
+      const accessToken = await getToken(CLI)
+      const { env: imsEnv = 'prod' } = await context.getCli()
+
+      try {
+        const generatedFile = 'console.json'
+        env.register(require.resolve('@adobe/generator-aio-console'), 'gen-console')
+        res = await env.run('gen-console', {
+          'destination-file': generatedFile,
+          'access-token': accessToken,
+          'ims-env': imsEnv
+        })
+        // trigger import
+        flags.import = generatedFile
+      } catch (e) {
+        console.log(chalk.red(e.message))
```

Comment:
not this.error() ? we don't exit, just continue?

Reply:
yup, as specified by @meryllblanchet